tests/contracts.hpp.in

```diff
@@ -14,16 +14,20 @@ struct contracts {
    static std::vector<char>    wrap_abi() { return read_abi("${CMAKE_BINARY_DIR}/../contracts/eosio.wrap/eosio.wrap.abi"); }
    static std::vector<uint8_t> bios_wasm() { return read_wasm("${CMAKE_BINARY_DIR}/../contracts/eosio.bios/eosio.bios.wasm"); }
    static std::vector<char>    bios_abi() { return read_abi("${CMAKE_BINARY_DIR}/../contracts/eosio.bios/eosio.bios.abi"); }
+   static std::vector<uint8_t> sendinline_wasm() {return read_wasm("${CMAKE_BINARY_DIR}/../contracts/test_contracts/sendinline/sendinline.wasm"); }
+   static std::vector<char>    sendinline_abi() {return read_abi("${CMAKE_BINARY_DIR}/../contracts/test_contracts/sendinline/sendinline.abi"); }
+   static std::vector<uint8_t> wrongcon_wasm() {return read_wasm("${CMAKE_BINARY_DIR}/../contracts/test_contracts/sendinline/wrongcon.wasm"); }
```

Comment:
There is no need for the `wrongcon` contract. It is identical to `sendinline` so just use that.

Reply:
Removed file `wrongcon.cpp`.